src/components/Swiper/Swiper.tsx

```diff
@@ -324,14 +325,22 @@ export const Swiper = (inProps: SwiperProps) => {
   const setActiveSlideByStep = useCallback(
     (step: number, options?: { smooth?: boolean }) => {
       if (container.current) {
+        if (active === to && step === 1 && loop) {
+          container.current.scrollTo({ [mapping.start]: 0, behavior: 'smooth' });
+          return;
+        }
+        if (active === from && step === -1 && loop) {
+          container.current.scrollTo({ [mapping.start]: getStep(1, to), behavior: 'smooth' });
```

Comment:
Так наверно попроще для восприятия будет
```suggestion
          container.current.scrollTo({ [mapping.start]: container.current[mapping.scrollSize], behavior: 'smooth' });
```